app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split(" ")
+    if files[1] != files[2]:
```

Comment:
How does your code handle a case if there's wrong command, not cp

Reply:
Didn't admit that it was necessary. But there's np to implement stuff like this.

Reply:
"Make sure you check that the `cp` input command is correct."
You're right, btw.)